main.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/DataDog/test-infra-definitions/registry"
+	_ "github.com/DataDog/test-infra-definitions/registry/scenarios"
+
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi/config"
+)
+
+const (
+	scenarioEnvVarName = "PULUMI_SCENARIO"
+	scenarioParamName  = "scenario"
+
+	dummyScenario = "dummy"
+)
+
+func main() {
+	pulumi.Run(func(ctx *pulumi.Context) error {
+		scenarioName := os.Getenv(scenarioEnvVarName)
+		rootConfig := config.New(ctx, "")
+		if s := rootConfig.Get(scenarioParamName); s != "" {
+			scenarioName = s
+		}
+
+		if scenarioName == dummyScenario {
+			return nil
+		}
+
+		rf := registry.Scenarios.Get(scenarioName)
+		if rf == nil {
+			return fmt.Errorf("impossible to run unknown scenarios: %s", scenarioName)
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
we could print available scenarios here

Reply:
Good idea, implemented!